src/components/container/AIAssistant/AIAssistant.tsx

```diff
@@ -89,14 +84,27 @@ export default function (props: AIAssistantProps) {
                 }
                 else if (type === "end") {
                     setLoading("");
+
+                    prettier.format(streamEvent.name + "(" + streamEvent.data.input.input + ")", { parser: "babel", plugins: [parserBabel, prettierPluginEstree] })
```

Comment:
Minor, but for readability these values could be put in variables like `toolName` and `toolInput`.